cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -270,7 +270,7 @@ class EvaluatorCommand : OrtCommand(
             ortResultInput = ortResultInput.replaceConfig(config)
         }
 
-        if (packageConfigurationOption != null) {
+        if (packageCurationsDir != null || packageCurationsFile != null) {
```

Comment:
Actually, that's how @fviernau had it before [my review comment](https://github.com/oss-review-toolkit/ort/pull/5994#discussion_r1005498327). Does the current code not work? Because if it does, I still believe it reads nicer to emphasize that you want to do something as soon as any of the mutually exclusive options is specified.

Reply:
The current code is wrong, `packageConfigurationOption` is for `PackageConfiguration` providers, but this code is about replacing `PackageCuration`s. For those there is no `packageCurationOption`, just the two separate options.

Reply:
Ah, right, a *configuration* vs. *curation* mixup.